lib/src/liveapp.ts

```diff
@@ -2,6 +2,7 @@
  * LiveApp query params.
  */
 export const QueryParams = {
+  Provider: "provider",
```

Comment:
The `providerId` should not be sent by LedgerLive. This is a setting that should be set by the provider's dev team.

Reply:
Not sure about this. I do not think we must supposed we are going to swap always with Changelly.

Reply:
I'd prefer not to expose it (the "provider" param) through the SDK, as the `QueryParams` is in the /lib dir.
We can define it in the /example/src dir.

Reply:
changed!